frontend/packages/shared/defi/lib/utils.ts

```diff
@@ -0,0 +1,56 @@
+
+import moment from "moment";
+
+export const SECONDS = 1 * 1000;
+export const MINUTES = 60 * SECONDS;
+export const HOURS = 60 * MINUTES;
+/** 1 Day in milliseconds */
+export const DAYS = 24 * HOURS;
+
+export type Period = {
+    days?: number;
+    years?: number;
+    months?: number;
+    weeks?: number;
+};
+
+export function createDurationPresetLabel(period: Period): string {
+    let suf = period.days
```

Comment:
Maybe a switch / case or some ifs here would be more readable ?

Reply:
@andresmechali  cleaned